Sources/Apodini/Request/Request.swift

```diff
@@ -6,29 +6,151 @@
 //
 import Foundation
 import NIO
-import protocol Fluent.Database
+import protocol FluentKit.Database
+import struct FluentKit.DatabaseID
+
+protocol ApodiniRequest: CustomStringConvertible, CustomDebugStringConvertible {
```

Comment:
Why did you rename `Request` to `ApodiniRequest`?

Reply:
I made `Request` a struct and needed a name for the protocol, and that's what I came up with. I didn't want to use something like `AnyRequest`. But happy for any naming suggestions.

Reply:
As the protocol is "public internal" API I exchanged the naming. Via e2981f5